bigquery_etl/docs/mozfun/generate_mozfun_docs.py

```diff
@@ -88,6 +113,26 @@ def generate_mozfun_docs(out_dir, project_dir):
                             dataset_doc_file.write(f"{formated}\n\n")
                         # Inject the contents of the README.md
                         dataset_doc_file.write(docfile_content)
+                        if is_udf:
+                            input_lines, output_lines = get_inputs_outputs_from_udf(
+                                root
+                            )
+
+                            # assemble, format, write inputs and outputs
+                            if len(input_lines) > 0:
```

Comment:
It's great to have the len() validation. I think we could simplify a lot the code by printing the parameters as returned in get_udf_parameters, (and moving any required logic there), wdyt?